src/components/Tooltip/components/TooltipOverlay/TooltipOverlay.tsx

```diff
@@ -12,6 +12,7 @@ export interface TooltipOverlayProps {
   id: string;
   active: boolean;
   light?: boolean;
+  preventInteraction?: boolean;
```

Comment:
Personally, I would write this as:
`preventInteraction?: PositionedOverlayProps['preventInteraction'];`